src/test/java/tests/SearchByNameTest.java

```diff
@@ -0,0 +1,27 @@
+package tests;
+
+import base.TestBase;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.PhonesPage;
+import utilities.LoadProperties;
+
+public class SearchByNameTest extends TestBase {
+
+    private  HomePage homeObj;
+    String macProduct = LoadProperties.productsSearchData.getProperty("Mac");
+
+    @BeforeMethod
+    public void beforeMethod(){
+        homeObj = new HomePage(driver);
+    }
+
+
+    @Test
+    public void searchByNameForMac(){
+        homeObj.searchForMac(macProduct);
+        Assert.assertTrue(homeObj.MacListOfSearch.contains("MacBook"));
```

Comment:
leave a failure message, something like this:
Assert.assertTrue(validationPageObj.isPasswordValidationErrorDisplayed(),"Password Validation Error was not displayed");